ouroboros-consensus-shelley-test/src/Test/ThreadNet/TxGen/Shelley.hs

```diff
@@ -107,7 +108,9 @@ genTx _cfg slotNo TickedShelleyLedgerState { tickedShelleyLedgerState } genEnv =
     ledgerEnv :: SL.LedgerEnv (MockShelley h)
     ledgerEnv = SL.LedgerEnv {
         ledgerSlotNo   = slotNo
-      , ledgerIx       = 0 -- TODO Ix
+      -- The use of 'mkTxIxPartial' is acceptible in test code,
+      -- especially below where we know that 0 is in the domain.
+      , ledgerIx       = SL.mkTxIxPartial 0
```

Comment:
This way there is no need for a comment ;) 
```suggestion
      , ledgerIx       = minBound
```

Reply:
much better, thank you @lehins